OsmAnd/src/net/osmand/plus/views/layers/MapTileLayer.java

```diff
@@ -71,21 +78,25 @@ public void initLayer(@NonNull OsmandMapTileView view) {
 		settings = view.getSettings();
 		resourceManager = view.getApplication().getResourceManager();
```

Comment:
It's already moved in the scope of PR #13673 , thus better to avoid duplication of the changes (like in this case for instance) in the different branches to minimize risks of possible merge conflicts.